indexer/db/withdrawal.go

```diff
@@ -40,6 +40,8 @@ type WithdrawalJSON struct {
 	TxHash                   string          `json:"transactionHash"`
 	Batch                    *StateBatchJSON `json:"batch"`
 	BedrockWithdrawalHash    *string         `json:"bedrockWithdrawalHash"`
+	BedrockProvenTxHash      *string         `json:"bedrockProvenTxHash"`
+	BedrockProvenLogIndex    *int            `json:"bedrockProvenLogIndex"`
```

Comment:
Noob question, what is a log index?

Reply:
The log index is the index of the event within the block- for ex., if there are 3 events emitted in the block and the `ProvenWithdrawal` is the 3rd, the log index would be `2`.